x-pack/plugins/security_solution/cypress/integration/cases/resolve.spec.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getCase1 } from '../../objects/case';
+
+import {
+  CASE_DETAILS_PAGE_TITLE,
+  CASE_CONFLICT_CALL_OUT_DISMISS,
+  CASE_CONFLICT_CALL_OUT_REDIRECT,
+} from '../../screens/case_details';
+
+import { goToCaseDetails } from '../../tasks/all_cases';
+import { createCase } from '../../tasks/api_calls/cases';
+import { dismissConflict, redirectConflict } from '../../tasks/case_details';
+import { cleanKibana } from '../../tasks/common';
+import { loginAndWaitForPageWithoutDateRange } from '../../tasks/login';
+
+import { CASES_URL } from '../../urls/navigation';
+
+describe('Resolve Cases', () => {
+  beforeEach(() => {
+    cleanKibana();
+    createCase(getCase1()).then((response) => {
+      cy.wrap({
+        ...getCase1(),
+        id: response.body.id,
+      }).as('myCase');
+    });
+  });
+
+  it('Resolves the case with exactMatch', function () {
+    loginAndWaitForPageWithoutDateRange(CASES_URL);
+    goToCaseDetails();
+
+    cy.url().should('include', `cases/${this.myCase.id}`);
+
+    cy.get(CASE_DETAILS_PAGE_TITLE).should('have.text', this.myCase.name);
+    cy.get(CASE_CONFLICT_CALL_OUT_DISMISS).should('not.exist');
+    cy.get(CASE_CONFLICT_CALL_OUT_REDIRECT).should('not.exist');
+  });
+
+  // TODO: implement createAliasCase function and remove skip
```

Comment:
Todo

Reply:
I missed that. we decided this test is not needed anymore